test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -34,10 +64,19 @@ export default class PossibleIssues extends Component {
             for (let index = 0; index < relatedIssues.items.length; index++) {
                 const repoName = relatedIssues.items[index].repository_url.replace(repoUrlAPIPrefix, "");
                 const issue = <a href={relatedIssues.items[index].html_url} target="_blank" rel="noopener noreferrer">{relatedIssues.items[index].title}</a>;
+
+                let relatedDegree = 'Medium';
+                if (repoName.includes(mlIssueRepo)) {
+                    if (relatedIssues.items[index].number.toString() === mlIssueNum) {
+                        relatedDegree = 'High';
+                    }
+                }
```

Comment:
Since possible issues have already been selected to contain the target test name, so I set the default degree to "Medium", if ML predicts its content is more related, then we set it as "High".